website/docs/ref/extractor-vue.md

```diff
@@ -0,0 +1,32 @@
+# Vue.js extractor
+
+The `@lingui/extractor-vue` package provides a [custom extractor](../guides/custom-extractor.md) that handles Vue.js files.
```

Comment:
```suggestion
The `@lingui/extractor-vue` package provides a [custom extractor](/docs/guides/custom-extractor.md) that handles Vue.js files.
```

Reply:
also, I would suggest moving this article to the tutorials section + adding the "## Further reading" section with a link to the "Custom Extractor" article